wom/src/main/scala/wom/expression/InputPointerToWomValue.scala

```diff
@@ -21,14 +21,22 @@ object InputPointerToWomValue extends Poly1 {
   implicit def fromOutputPort: Case.Aux[OutputPort, ToWdlValueFn] = at[OutputPort] {
     port => (knownValues: Map[String, WomValue], ioFunctions: IoFunctionSet, outputPortLookup: OutputPortLookup, inputDefinition: InputDefinition) =>
       (outputPortLookup(port), inputDefinition) match {
-        case (v: Valid[WomValue], _) => v.map(inputDefinition.valueMapper(ioFunctions))
+        case (Valid(womValue), _) if isDefined(womValue) =>
```

Comment:
Just curious about the addition of `ifDefined`

Reply:
Conformance test \#15 was failing because `outputPortLookup(port)` was returning a `Valid(WomOptionalValue(WomAnyType, None))` that is effectively `null`/`None`. Without the `isDefined`, this case statement on line 24 was matching, when instead the next case statement below should have been matching and using the `default` value provided within the CWL.